components/portal/widgets/controllers.js

```diff
@@ -671,6 +671,8 @@ define(['angular', 'moment'], function(angular, moment) {
               eval($scope.widget.widgetConfig.evalString)) {
               $scope.isEmpty = true;
             }
+          } else if (($scope.widget === 'student-orientation') && !data) {
```

Comment:
Clever. I'm fine with this as a one-off. 

Nonetheless, there are tradeoffs here.

This is coding into the framework configuration specific to a single app directory entry. If we need this feature for more than one entry, it might be worth it to refactor it into the framework as a shared feature, as in error configuration or an error template or so goes into the data model for app directory entries and any entry that declares a custom error template can use this feature.

So. :shipit: . But when it becomes annoying to have to cut a framework release to edit this error message or when we do to add another widget id to this conditional, let's refactor. :)

Reply:
Aside: this is making me think about, what if we embraced this approach, and merged all the app directory data into the framework. Stop trying to store that in and read it from a database. Just code. Might be worth it for easier testing and reducing runtime dependency.

Reply:
Thank you for reviewing Andrew! 
I've been thinking about this solution, and I agree with you that there are more efficient ways to accomplish this. I'm going to close this MR, and move forward with [handling the error message in the entity file](https://git.doit.wisc.edu/myuw-overlay/entities/-/merge_requests/2726).